libfwupdplugin/fu-security-attrs.c

```diff
@@ -119,14 +120,24 @@ fu_security_attrs_append(FuSecurityAttrs *self, FwupdSecurityAttr *attr)
  * Since: 1.7.2
```

Comment:
as this function is changing arguments, it should be updated to 1.9.6?

Reply:
Ya, probably -- I've updated README too. Fixed, thanks.